.github/actions/build_pandas/action.yml

```diff
@@ -22,6 +22,13 @@ runs:
         fi
       shell: bash -el {0}
 
+    - name: Uninstall nomkl
+      run: |
+        if conda list nomkl | grep nomkl 1>/dev/null; then
+          conda remove nomkl -y
+        fi
+      shell: bash -el {0}
```

Comment:
@mroeschke do you remember why this was needed? (I don't see anything in the PR description or discussion)